python-modules/chevah-python-test/test_python_binary_dist.py

```diff
@@ -421,7 +421,7 @@ def main():
 
             if chevah_os == 'windows':
                 # Check OpenSSL version on windows.
-                expecting = u'OpenSSL 1.0.2g  1 Mar 2016'
+                expecting = u'OpenSSL 1.1.0f  25 May 2017'
```

Comment:
it should be fine if we have different versions.

we are using stdlib OpenSSL only for sha hash operations.

All TLS/SSL and serious crypto stuff is done via Python cryptography

--------

but I don't know if our server code can handle all OpenSSL 1.1.0 changes... in the case in which there are backward incompatible changes.